scripts/solutions.sh

```diff
@@ -138,6 +138,10 @@ while :; do
     case $1 in
         -a|--archive)
             shift
+            if [ ! -e "$1" ]; then
+                echo 1>&2 "No such file or directory '$1'"
+                exit 1
+            fi
             ARCHIVES+=("$(readlink -f "$1")")
```

Comment:
What if the link exists but doesn't resolve? Is `test -e` checking this?

Reply:
You mean if the link points to non-existing file or directory ?
The check fails.